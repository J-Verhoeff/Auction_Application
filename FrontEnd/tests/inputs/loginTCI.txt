logout
login
user1
login
admin
random
logout
login
user
create
logout
bid
exit