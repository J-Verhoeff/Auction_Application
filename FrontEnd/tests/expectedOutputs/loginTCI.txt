
Welcome to the Auction App!
Start entering transactions
To exit the application type 'exit'

Options:
	login
Transaction: Transaction not in options!


Options:
	login
Transaction: User: User not in database!


Options:
	login
Transaction: User: 
Welcome admin!
User Type: AA
Available Credit: 100.00$

Options:
	logout
	create
	delete
	advertise
	bid
	refund
	addcredit
Transaction: Transaction not in options!


Options:
	logout
	create
	delete
	advertise
	bid
	refund
	addcredit
Transaction: 
Welcome to the Auction App!
Start entering transactions
To exit the application type 'exit'

Options:
	login
Transaction: User: 
Welcome user!
User Type: FS
Available Credit: 0.00$

Options:
	logout
	advertise
	bid
	addcredit
Transaction: Transaction not in options!


Options:
	logout
	advertise
	bid
	addcredit
Transaction: 
Welcome to the Auction App!
Start entering transactions
To exit the application type 'exit'

Options:
	login
Transaction: Transaction not in options!


Options:
	login
Transaction: Goodbye!
